docsrc/markdown/application_service.md

```diff
@@ -0,0 +1,34 @@
+# Application Service
+
+The Application Service is a component at the service layer of the BDK which aims to cover the Applications part of the [REST API documentation](https://developers.symphony.com/restapi/reference).
+More precisely:
+* [Create application](https://developers.symphony.com/restapi/reference#create-app)
+* [Create application with an RSA public key](https://developers.symphony.com/restapi/reference#create-application-with-an-rsa-public-key)
+* [Update application](https://developers.symphony.com/restapi/reference#update-application)
+* [Update application with an RSA public key](https://developers.symphony.com/restapi/reference#update-application-with-an-rsa-public-key)
+* [Delete application](https://developers.symphony.com/restapi/reference#delete-application)
+* [Get application](https://developers.symphony.com/restapi/reference#get-application)
+* [List application entitlements](https://developers.symphony.com/restapi/reference#list-app-entitlements)
+* [Update application entitlements](https://developers.symphony.com/restapi/reference#update-application-entitlements)
+* [List user applications](https://developers.symphony.com/restapi/reference#user-apps)
+* [Update user applications](https://developers.symphony.com/restapi/reference#update-user-apps)
+
+
+## How to use
+The central component for the Application Service is the `ApplicationService` class.
+This class exposes the user-friendly service APIs which serve all the services mentioned above 
+and is accessible from the `SymphonyBdk` object by calling the `applications()` method:
+```python
+class ApplicationMain:
+    @staticmethod
```

Comment:
would a simple script be more pythonic? (i.e no class just a method)

Reply:
I let it as a class to be consistent with the documentations for other services